.github/workflows/daily-i18n.yaml

```diff
@@ -0,0 +1,30 @@
+name: daily-i18n
```

Comment:
Is this still needed in this repo or would it be moved to a separate repo?

Reply:
Oops! I am will remove this action. 

By the way did you consider, transfer xingxing/renovate-i18n to renovatebot ?